Orange/widgets/visualize/owdistributions.py

```diff
@@ -16,13 +15,7 @@
 from Orange.widgets.utils import itemmodels, colorpalette
 from Orange.widgets.widget import InputSignal
 
-
-def is_discrete(var):
-    return isinstance(var, Orange.data.DiscreteVariable)
-
-
-def is_continuous(var):
-    return isinstance(var, Orange.data.ContinuousVariable)
+import pyqtgraph as pg
```

Comment:
Ah, yes, because the modules didn't exec for me with that import order. In fact, I get this inside virtualenv:
```python
>>> import pyqtgraph
>>> import Orange.widgets
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jk/Downloads/orange3/Orange/widgets/__init__.py", line 4, in <module>
    import pkg_resources
  File "/usr/lib/python3/dist-packages/pkg_resources.py", line 50, in <module>
    from cStringIO import StringIO as BytesIO
ImportError: No module named 'cStringIO'
```
The culprits are [pyqtgraph setting `basestring` as builtin in Python3](https://github.com/pyqtgraph/pyqtgraph/blob/1ecceeaa9248c180133b8a52edf7ada29b43931c/pyqtgraph/python2_3.py#L44) and pkg_resources relying on `basestring` existing only in Python 2.

Upgrading to the latest setuptools resolved it for me. I shall revert those import changes.